selection/src/commonMain/kotlin/Group.kt

```diff
@@ -0,0 +1,8 @@
+package com.juul.krayon.selection
+
+import com.juul.krayon.element.Element
+
+public data class Group<E : Element, D>(
```

Comment:
Is `D` the generic for "data type" or datum?

Reply:
`D` is the type of data stored in the elements (or `Nothing?`, if you haven't called `.data` yet).

Reply:
> Not really sure what distinction you're drawing between data type and datum here.

I had misremembered the meaning of "datum", which after looking it up, I realize is essentially is "data", so they're essentially the same thing.

You can disregard that portion of my comment. And you answered the other portion of the comment. Thank you!